src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -488,8 +488,13 @@ bool DefaultPolicy::BudgetCheck() const
     // The strategy tracks the amout of inlining done so far,
     // so it performs the actual check.
     //
-    InlineStrategy* strategy   = m_RootCompiler->m_inlineStrategy;
-    const bool      overBudget = strategy->BudgetCheck(EstimatedTotalILSize());
+    InlineStrategy* strategy = m_RootCompiler->m_inlineStrategy;
+
+    const bool preferSpeed = m_RootCompiler->opts.jitFlags->IsSet(JitFlags::JIT_FLAG_SPEED_OPT);
+    const bool isAot       = m_RootCompiler->opts.IsReadyToRun();
```

Comment:
I see this elsewhere:
```
    const bool isPrejit   = compiler->opts.jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT);
    const bool isSpeedOpt = compiler->opts.jitFlags->IsSet(JitFlags::JIT_FLAG_SPEED_OPT);
```

should it be the same here? (i.e., JIT_FLAG_PREJIT instead of IsReadyToRun() / JIT_FLAG_READYTORUN)